File: 1386. Bafo/BAFO.java
import java.util.Scanner;

class BAFO{
	public static void main(String[] args){
		int acmA, acmB, nRodadas, teste;
		Scanner entrada = new Scanner(System.in);

		nRodadas = entrada.nextInt();
		teste = 1;
		while(nRodadas > 0){
			acmA = 0;
			acmB = 0;
			
			for(int i = 0; i < nRodadas; i++){
				acmA += entrada.nextInt();
				acmB+= entrada.nextInt();
			}

			System.out.println("Teste " + teste);
			if(acmB > acmA)
				System.out.println("Beto\n");
			else
				System.out.println("Aldo\n");

			teste++;
			nRodadas = entrada.nextInt();

		}

	}
}

File: 1386. Bafo/bafo.c
#include <stdio.h>

int main(){
	int acmA, acmB, nRodadas, teste;
	int i, aux;

	scanf("%d", &nRodadas);
	teste = 1;
	while(nRodadas > 0){
		acmA = 0;
		acmB = 0;
		
		for(i = 0; i < nRodadas; i++){
			scanf("%d", &aux);
			acmA += aux;
			scanf("%d", &aux);
			acmB += aux;
		}

		printf("Teste %d\n", teste);

		if(acmA > acmB)
			printf("Aldo");
		else
			printf("Beto");

		printf("\n\n");

		scanf("%d", &nRodadas);
		teste++;
	}

	return 0;
}
